changelog/fragments/bump-helm-operator-plugins.yaml

```diff
@@ -0,0 +1,16 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Bump the helm-operator-plugins dependency version to v0.0.10.
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "change"
```

Comment:
is this getting rolled into description or is just highlighted weird here?

Reply:
From what I can tell locally it matches the template file so this seems like it isn't getting rolled into the description, so I think it is likely just a weird highlight